.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ on:
     paths-ignore:
       - 'docs/**'
       - '*.md'
-  pull_request_target:
+  pull_request:
```

Comment:
Why is this file changed?

Reply:
ref: https://github.com/fastify/fastify/pull/3741. And I rebased from master